nixos/modules/virtualisation/waydroid.nix

```diff
@@ -36,6 +36,10 @@ in {
       (isEnabled "ASHMEM")
     ];
 
+    boot.kernelParams = mkIf
+      (config.lib.kernelConfig.isEnabled "CONFIG_PSI_DEFAULT_DISABLED")
+      [ "psi=1" ];
+
```

Comment:
Reading the kernelConfig is not always possible.

It might be preferable to always add `psi=1` when waydroid is enabled.

This is because `psi=1` would be a no=op if the feature is not enabled in the kernel. If it is ***not** default disabled*, it would be a no-op too I presume.